### PR TITLE
Faranak

### PR DESCRIPTION
**Pull request by Faranak36**
review please

### CODE REVIEW

File: src/faranak/week2/DivisibleNumbers.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package faranak.week2;
+
+public class DivisibleNumbers {
+
+    public static void main(String[] args) {
```
[the remark this answers is not in the document]
Comment (Faranak36) [reply src/faranak/week2/DivisibleNumbers.java:5]: But I double checked it after the meeting ,it's exactly the same output that task is asking for.check the output in the task.

[the remark this answers is not in the document]
Comment (Faranak36) [reply src/faranak/week2/DivisibleNumbers.java:5]: Ok I just saw the mistake, will fix it

File: src/faranak/week2/ConsecutiveNumbers.java
Comment (cbanavi) [inline src/faranak/week2/ConsecutiveNumbers.java:1]: lgtm

File: src/faranak/week2/DivisibleNumbers.java
Comment (cbanavi) [inline src/faranak/week2/DivisibleNumbers.java:1]: lgtm

File: src/faranak/week2/SwapNumbers.java
Comment (cbanavi) [inline src/faranak/week2/SwapNumbers.java:1]: lgtm

**Review by cbanavi — APPROVED** [review]
Nicely done!

**Pull request merged**